### ISSUE TITLE
Enable/Disable Sequencer 

### ISSUE THREAD

**Issue by mhabimana** [opencdms/Climsoft#517]
At the moment it is not possible to disable the sequencer on the data entry form  permanently because the sequencer is enabled by default and by clicking on the check box in front of the sequencer label,the sequencer is disabled temporary but when you close the form and reopen this again the sequencer become enabled. I think the best way is to give users an option to enable or disable the sequencer  permanently.

**Comment by smachua** [comment]
@mhabimana I will work on it.

**Comment by Patowhiz** [comment]
@mhabimana @smachua @isedwards  should this be implemented in in all the data entry forms? If yes, then I suggest we have this option inside what we discussed during the April workshop. An additional button to be introduced in the forms, this button could be label "More Options", then we can have it pop up a custom context menu or a sub-dialog in which it gives such an option and many more. And if it's going to be implemented as a setting, should it be a user based setting or a general setting and where should it be saved, in the database or the registry?

**Comment by mhabimana** [comment]
@Patowhiz - I leave this to @smachua to elaborate more on this.

**Comment by mhabimana** [comment]
@smachua - This needs your attention.

**Comment by Patowhiz** [comment]
@mhabimana if it's agreed that this setting should come from the database, I think it's much better to have it in version 5 model. I can try to have added in the prototype as part of the demonstration of version 5 model.

**Comment by isedwards** [comment]
Should this be an organisation-wide setting in the central database that applies to all users? If yes,
a) only an administrator should be able to change the default setting for everyone
b) we should probably continue to allow users to _temporarily_ change their setting, as they do now, if they need to (in case it is ever set incorrectly and they need to switch the sequencer on manually).

I think small changes like this can be made to the version 4 data model (it looks like we could just store the preference in the existing  `regkeys` table?)

**Comment by smachua** [comment]
I agree with @isedwards on (a), (b) and that these changes be implemented in V4 data model. The existing table 'data_forms' is the most suitable for this purpose since it contains details of all forms. An attribute can be added to hold information on forms sequencer status. The modification may be implemented through a code rather than SQL script.

**Comment by isedwards** [comment]
For changes like this, it makes sense to use "progressive enhancement" in the code.

If the database schema contains the new attribute then the code uses it, otherwise the error is caught and we fall back to just using the current hard coded default.
